### ISSUE TITLE
Sign up page title is not meaningful

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4000]
## Area or Component
Sign up form

## Quick Summary
The page titles across the sign up experience are just "SimpleReport", which is not descriptive enough.

Ideally, we would have different titles for each page of the sign up experience (choosing the sign up, the organization information form, the Experian agreement, the personal details form, the Experian questions, and the final success/failure message)

## Accessibility Violation
Page <title> does not identify purpose of page: https://cdc-devtools.dequecloud.com/rule-help/en/title-not-meaningful

## Action Requested
Edit the page titles for the sign up experience to be more meaningful.

## Original Deque Issue
title-not-meaningful | The <title> element on this page does not identify the contents or purpose of the page. | Title of the page is SimpleReport but would more accurately be SimpleReport \| Sign up or similar. | Page <title> does not identify purpose of page | https://cdc-devtools.dequecloud.com/rule-help/en/title-not-meaningful | serious | Yes | No | Structure | ["title"] | The <title> element on this page does not identify the contents or purpose of the page. | <title>SimpleReport</title> | wcag2a,wcag242a | rnv7@cdc.gov | Sign up selector | https://www.simplereport.gov/app/sign-up/ |   | 2022-07-15T19:15:36.268Z
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --

**Comment by lauraykerr** [comment]
Based on existing pattern for titles, recommend we go with `Sign up | SimpleReport`.

**Comment by emmastephenson** [comment]
@lauraykerr Would you recommend that we use `Sign up | SimpleReport` for all pages throughout the signup flow? There are a couple pages users will go through:

1. The selector screen
![Screen Shot 2022-07-26 at 2 59 12 PM](https://user-images.githubusercontent.com/80282552/181119866-01ea90ca-0174-46a4-a028-ae2080d40498.png)

2. The Organization information screen
![Screen Shot 2022-07-26 at 2 59 18 PM](https://user-images.githubusercontent.com/80282552/181119749-4394d237-2975-4d52-b297-c120ba99d4a3.png)

3. The Experian agreement
![Screen Shot 2022-07-26 at 2 55 57 PM](https://user-images.githubusercontent.com/80282552/181119319-eee995fa-1d8e-47a6-8ae8-a19d86b444bf.png)

4. The personal details
![Screen Shot 2022-07-26 at 2 58 45 PM](https://user-images.githubusercontent.com/80282552/181119682-762dd382-ec33-4db5-a52f-a4eab474f0b9.png)

5. The Experian questions
6. The Experian response (success/failure)

**Comment by lauraykerr** [comment]
@emmastephenson Ah yes, that makes sense — thanks for clarifying and including these screenshots! For the different pages, would recommend: 

1. Sign up - select status | SimpleReport 
2. Sign up - organization information | SimpleReport 
3. Sign up - identity verification consent | SimpleReport 
4. Sign up - personal details | SimpleReport 
5. Sign up - identity verification | SimpleReport 
6A. Sign up - success | SimpleReport
6B. Sign up - schedule call | SimpleReport

**Comment by lauraykerr** [comment]
@emmastephenson Punting this back to eng backlog to include with signup work.

**Comment by emyl3** [comment]
Another is if you click "I'm trying to get my COVID-19 test results"
<img width="597" alt="Screen Shot 2022-08-09 at 10 02 17 AM" src="https://user-images.githubusercontent.com/20211771/183685903-3a7abaf5-ac05-4578-a795-85c7882e1016.png">

the user gets redirected to
<img width="660" alt="Screen Shot 2022-08-09 at 10 02 23 AM" src="https://user-images.githubusercontent.com/20211771/183685985-157ef28a-107c-4f62-9423-920b4442919a.png">
Let me know what this page title should be as well!

Thank you very much!

**Comment by lauraykerr** [comment]
Thanks @emyl3! I should have tested the flow more thoroughly. For this page, recommend `Request access | SimpleReport` <img alt="Screen Shot 2022-08-09 at 9 59 39 AM" width="741" src="https://user-images.githubusercontent.com/20211771/183685521-bbad196b-da9a-46ad-bab0-00d0ce6b3718.png">

**Comment by lauraykerr** [comment]
For this, thinking `Get COVID-19 test results | SimpleReport`. Thanks @emyl3! 

<img alt="Screen Shot 2022-08-09 at 10 02 23 AM" width="660" src="https://user-images.githubusercontent.com/20211771/183685985-157ef28a-107c-4f62-9423-920b4442919a.png">
